lib/http2.c

```diff
@@ -47,6 +47,7 @@
 #include "curl_memory.h"
 #include "memdebug.h"
 
+#define DEBUG_HTTP2 1
```

Comment:
This won't affect any activity unless specifying VERBOSE option (or --verbose in command line).